services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -443,9 +444,9 @@ async def _save_state_folder(
 
 async def task_save_state(
```

Comment:
so this function saves all the defined state folderS right?
suggestion:

```suggestion
async def task_save_state_folders(
```